schema/google/showcase/v1beta1/identity.proto

```diff
@@ -58,7 +58,7 @@ service Identity {
   rpc UpdateUser(UpdateUserRequest) returns (User) {
     option (google.api.http) = {
       patch: "/v1beta1/{user.name=users/*}"
-      body: "*"
+      body: "user"
```

Comment:
Why not have `*` so that the user can specify which fields to PATCH? It seems like this is one of the prime uses cases for field masks.

Reply:
I'm not sure what you mean. The update_mask will end up in the query params. We don't lose any functionality here, just fixing a non-compliant binding https://google.aip.dev/134

Reply:
Ah, gotcha. "There must be a body key in the google.api.http annotation, and it must map to the resource field in the request message. All remaining fields should map to URI query parameters."

OK, my bad. LGTM!